Description: use Digest::SHA (in perl core) instead of Digest::SHA1
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2012-03-31

--- a/Build.PL
+++ b/Build.PL
@@ -25,7 +25,7 @@
         'Moose'        => 0,
         'Math::BigInt' => 0,
         'Digest::HMAC' => 0,
-        'Digest::SHA1' => 0,
+        'Digest::SHA' => 0,
     },
     add_to_cleanup     => [ 'Authen-OATH-*' ],
     create_makefile_pl => 'traditional',
--- a/META.yml
+++ b/META.yml
@@ -4,7 +4,7 @@
   - 'Kurt Kincaid <kurt.kincaid@gmail.com>'
 build_requires:
   Digest::HMAC: 0
-  Digest::SHA1: 0
+  Digest::SHA: 0
   Math::BigInt: 0
   Moose: 0
   Test::More: 0
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -6,7 +6,7 @@
   'VERSION_FROM' => 'lib/Authen/OATH.pm',
   'PREREQ_PM' => {
                    'Digest::HMAC' => 0,
-                   'Digest::SHA1' => 0,
+                   'Digest::SHA' => 0,
                    'Math::BigInt' => 0,
                    'Moose' => 0,
                    'Test::More' => 0,
--- a/lib/Authen/OATH.pm
+++ b/lib/Authen/OATH.pm
@@ -26,7 +26,7 @@
 has 'digest' => (
     'is'      => 'rw',
     'isa'     => 'Str',
-    'default' => 'Digest::SHA1'
+    'default' => 'Digest::SHA'
 );
 
 has 'timestep' => (
--- a/t/01-cases.t
+++ b/t/01-cases.t
@@ -12,7 +12,7 @@
 ok( defined $oath, "successfully created new object" );
 ok( $oath->isa( 'Authen::OATH' ), "correct class." );
 ok( $oath->{ 'digits' } == 6,     "default digits set to 6" );
-ok( $oath->{ 'digest' } eq 'Digest::SHA1', "default digest set to Digest::SHA1" );
+ok( $oath->{ 'digest' } eq 'Digest::SHA', "default digest set to Digest::SHA" );
 ok( $oath->{ 'timestep' } == 30, "default timestep set to 30" );
 
 print "Checking test vectors for totp()...\n";
